### DISCUSSION TITLE
Keycloak Time zones

### DISCUSSION THREAD

**Discussion by minime4568** [keycloak/keycloak#17400] [Q&A]
I am new to keycloak, Is there any feature in keycloak where we can change the time zones or is it by default linked to the machine time zone where keycloak is installed. 
Keycloak version I am using is 20.0.3
Thanks in advance.

**Comment by meveno** [comment]
You can set this at the JVM level.
Example :
`java -Duser.timezone="Europe/Paris"`

It depends on the way you have deployed keycloak but usually it can be done by defining an environment variable like JAVA_OPTS like this :
`JAVA_OPTS=java -Duser.timezone="Europe/Paris"`

For available timezones, see [Timezones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List)
